frontend/utils.py

```diff
@@ -290,8 +290,15 @@ def train_df(df, min_edge=0, rank=False):
 
 def rule_chooser():
     option = st.selectbox("Choose from the rules", st.session_state.sens)
-    G, _ = default_pn_to_graph(option.split(";")[0])
-    text_G, _ = default_pn_to_graph(option.split(";")[0])
+    first_option = option.split(";")[0]
```

Comment:
Can you please provide an example for a pattern that it would match? 
Does this matcher always apply, or do I have to call POTATO in a specific way?

Reply:
This would match any penman as well as any command given with the command(penman1, penman2) formula.
For example, the following rule:
 > path((u_800 / like :nsubj (u_1 / I)),(u_120 / flower))

Would match any graph where the two subgraphs are present and there is a directed path from 'like' to 'flower'.
Currently, there are 3 commands supported: path, undirected and any integer (e.g. 2 and 4 are valid commands).

I've set this as the default, but we can obviously change it.